### PR TITLE
fix: removed unnecessary toasts in faq activity

### LINKED ISSUE #586: Bug:  Unnecessary Toast In FAQ Section
**Summary:**
On clicking the list items of the FAQ section, a toast message is being displayed.

**Steps to reproduce:**
FAQ -> Click on any Question

**Expected behavior:**
No toast should be shown as it does not look good from the eyes of the user.

**Observed behavior:**
A toast is displayed with some irrelevant text in it.

**Device and Android version:**
Android 9, Mi A2
**Screenshots:**
issue

### PR DESCRIPTION
**Pull request by rachittt**
Fixes #586 

Screenshot:
ezgif com-video-to-gif (7)

Please Add Summary of what changes you have made.(Optional)

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by rachittt** [comment]
@luckyman20 can you please review this ?

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java [hunk 1]
```diff
@@ -74,30 +73,6 @@ public void initViews() {
      */
     public void initListeners() {
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java]: If there is no content left in the function, we can remove the function too.

Comment (rachittt) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java]: removed it. Thanks!

**Comment by Bhavnaharitsa** [comment]
@rachittt Does reforming again will solve Travis errors because I am constantly getting build failure :(

**Comment by rachittt** [comment]
`./gradlew check` always helps me :D

**Pull request merged**